metal3-templater.go

```diff
@@ -143,8 +144,8 @@ func main() {
 		// ProvisioningIP
 		size, _ := ipnet.IPNet.Mask.Size()
 		templateData.ProvisioningIP = fmt.Sprintf("%s/%d", clusterIP, size)
-		templateData.ClusterIronicURL = fmt.Sprintf("http://%s", net.JoinHostPort(clusterIP, "6385"))
-		templateData.ClusterInspectorURL = fmt.Sprintf("http://%s", net.JoinHostPort(clusterIP, "5050"))
+		templateData.ClusterIronicURL = fmt.Sprintf("https://%s", net.JoinHostPort(clusterIP, "6385"))
+		templateData.ClusterInspectorURL = fmt.Sprintf("https://%s", net.JoinHostPort(clusterIP, "5050"))
```

Comment:
Looking at CBO this will only work for 4.8 and onwards, I'm not sure that many folks are testing older versions locally now, but we might want to add a conditional if so (looking at https://access.redhat.com/support/policy/updates/openshift we may have to sometimes test 4.7/4.6EUS)